src/Pane.java

```diff
@@ -3,60 +3,59 @@
 import java.awt.*;
 
 public class Pane extends JLayeredPane {
-    final static int width = 640;
-    final static int height = 480;
-    Button one_player;
-    Button two_players;
+    final static int WIDTH = 640;
+    final static int HEIGHT = 480;
+    Button onePlayer;
+    Button twoPlayers;
     Button restart;
     Label label;
     JButton[][] fields;
-    int[] x_coordinates = new int[3];
-    int[] y_coordinates = new int[3];
-    final static Color player1_color = Color.orange;
-    final static Color player2_color = new Color(0, 150, 200);
-    final static Color game_over_field_color = Color.lightGray;
+    int[] xCoordinates = new int[3];
+    int[] yCoordinates = new int[3];
+    final static Color player1Color = Color.orange;
+    final static Color player2Color = new Color(0, 150, 200);
+    final static Color gameOverFieldColor = Color.lightGray;
     // First panel constructor
     Pane(){
-        standard_settings();
-        one_player = new Button("One player");
-        one_player.setLocation((int)(width-2*one_player.getWidth())/3, (int)(height-one_player.getHeight())/2);
-        one_player.setOpaque(true);
-        this.add(one_player);
+        standardSettings();
+        onePlayer = new Button("One player");
```

Comment:
chyba bym zmienił to na onePLayerButton żeby od razu było wiadomo że to jest instancja klasy Button (nie znalazłem nic o tym w necie, ale tak mi jakoś pasuje chyba)